src/main/frontend/publishing/html.cljs

```diff
@@ -19,7 +19,7 @@
              :name "viewport"}]
            [:link {:type "text/css", :href "static/css/tabler-icons.min.css", :rel "stylesheet"}]
            [:link {:type "text/css", :href "static/css/style.css", :rel "stylesheet"}]
-           [:link {:type "text/css", :href "static/css/custom.css", :rel "stylesheet"}]
+           [:link {:type "text/css", :href "static/css/export.css", :rel "stylesheet"}]
```

Comment:
The only thing that concerns me is the backwards compatibility. Do you think it is a good idea to still adding `custom.css` here?

Reply:
It now defaults to export.css, but falls back to custom.css